reco_utils/recommender/geoimc/GeoIMCutils.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+from sklearn.decomposition import PCA
+
+from numba import jit
+from IPython import embed
+
+def length_normalize(matrix):
+    norms = np.sqrt(np.sum(matrix**2, axis=1))
+    norms[norms == 0] = 1
+    return matrix / norms[:, np.newaxis]
+
+
+def mean_center(matrix):
+    avg = np.mean(matrix, axis=0)
+    matrix -= avg
+
+
+def binarize(a, threshold):
+    return np.where(
+        a > threshold,
+        1.0,
+        0.0
+    )
```

Comment:
there is another function binarize function `binarize_rating`, it would be good to reuse.

Also, binarize looks like a general function. In this PR https://github.com/microsoft/recommenders/pull/112, @viktorku is also using binarization. Do you guys think we could have a binarize function in the common utils somewhere, that both PRs share?

Reply:
Resolved at b19a52b51b7df9dc272000623da4d82bfb7ceeff.

Yes, I think we should have it in common utils.